### PR TITLE
draft no review needed

### PR DESCRIPTION
**Pull request by steelehelbling**
## Bug description
Describe the problem

## Solution description
Describe the code change in detail for reviewer. Explain the technical solution you have provided and how it fixed the issue case. 

## Which file changed
Clarify which file changed, for example “Added Hello World to index.js”

## Covered unit test cases
Were unit test cases recorded for this fix?

## Tested this issue fix on all brower
[x] Chrome
[x] Safari
[x] Internet Explorer
[x] Firefox

## Keep Pull Request Simple
Pull Request is relatively small. It does not contain multiple changes or solve multiple problems. 

## Reviewed by Team members
Pull Request is reviewed by one or more team members 

## Idiomatic Code
Code follows industry standards for language, framework, and libraries. 

## Code is clear and maintainable:
- [ ] My code follows the style guidelines of this project
- [ ] My code is consistently formatted.
- [ ] No instance on duplicate code.

### CODE REVIEW

File: data/migrations/20200625220949_create-profile.js [hunk 1]
```diff
@@ -7,9 +7,34 @@ exports.up = (knex) => {
...
+      incident.date('date');
+    })
+    .createTable('sources', function (sources) {
+      sources.integer('incident_id').notNullable().unique().primary();
+      sources.integer('src_id');
+      sources.string('src_url');
+      sources.string('src_type');
+    })
+    .createTable('incident_type_of_force', function (incident_type_of_force) {
+      // incident_type_of_force.integer('itof_id').notNullable().unique().primary();
```
Comment (jasonelliots) [inline data/migrations/20200625220949_create-profile.js:28]: No commented out code in pull requests

**Review by jasonelliots — APPROVED** [review]
No comment

File: data/migrations/20200625220949_create-profile.js [hunk 2]
```diff
@@ -7,9 +7,34 @@ exports.up = (knex) => {
...
+    })
+    .createTable('type_of_force', function (type_of_force) {
+      // type_of_force.integer('type_of_force_id').notNullable().unique().primary();
+      type_of_force.string('type_of_force');
     });
 };
 
 exports.down = (knex) => {
-  return knex.schema.dropTableIfExists('profiles');
+  return knex.schema.dropTableIfExists('type_of_force').dropTableIfExists('incident_type_of_force').dropTableIfExists('sources').dropTableIfExists('incident').dropTableIfExists('profiles');
```
Comment (jasonelliots) [inline data/migrations/20200625220949_create-profile.js]: Test and publish test coverage failing here - look into this

**Review by jasonelliots — DISMISSED** [review]
No comment

**Review by emixo — APPROVED** [review]

**Pull request merged**